SPECS/cloud-init/cloud-init.spec

```diff
@@ -15,7 +15,7 @@ Patch1:         make_fallback_network_config_work.patch
 BuildRequires:  automake
 BuildRequires:  dbus
 BuildRequires:  iproute
-BuildRequires:  mariner-release 
+BuildRequires:  azurelinux-release 
```

Comment:
@microsoft/cbl-mariner-provisioning The distro renaming will have affects on how cloud-init behaves for us so we'll need to address this separately